apps/test/util/configuredChai.js

```diff
@@ -2,7 +2,7 @@
 import chai from 'chai';
 import chaiSubset from 'chai-subset';
 import chaiEnzyme from 'chai-enzyme';
-chai.use(chaiEnzyme);
+chai.use(chaiEnzyme());
```

Comment:
@pcardune added chai-enzyme in #9629 but didn't use it anywhere.  Turns out we were installing it incorrectly.  This is the correct approach, per the [chai-enzyme docs](https://github.com/producthunt/chai-enzyme#setup)